README.md

```diff
@@ -25,19 +36,19 @@ on:
 jobs:
   submit-graph:
     name: Submit Dependency Graph
-    runs-on: ubuntu-latest # or windows-latest, or macOS-latest
+    runs-on: ubuntu-latest
```

Comment:
I don't think we should even advertise other OS runners, there is no need to. Since this is a pretty small self-contained action, I think we can just say to use `ubuntu-latest` and call it a day. It will make everything easier.

Reply:
:+1: